src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js

```diff
@@ -3,10 +3,29 @@ import React from 'react';
 // Relative imports.
 import CallToActionWidget from 'platform/site-wide/cta-widget';
 import MoreInfoAboutBenefits from '../../../components/MoreInfoAboutBenefits';
+import CernerCallToAction from '../../../components/CernerCallToAction';
+
+const callToActions = [
+  {
+    deriveHeaderText: facilityNames =>
+      `View lab and test results from ${facilityNames}`,
+    href: '',
+    label: 'View results on My VA Health',
+  },
+  {
+    deriveHeaderText: () =>
+      `View lab and test results from another VA Medical Center`,
+    href: '',
```

Comment:
Need the 2 `href` links [here](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/405218216):

![image](https://user-images.githubusercontent.com/12773166/86204084-d92bd780-bb23-11ea-982c-43d27f012b37.png)

Reply:
**From Cerner for Mann-Grandstaff launch:**

Electronic Health Record Modernization (EHRM) Spokane Mann Grandstaff IOC: Veterans receiving care at VA Medical Centers who have converted to EHRM will have the following functionality:

View your VA lab and test results online
Link to this in Staging My VA Health: https://ehrm-va-test.patientportal.us.healtheintent.com/pages/health_record/results/labs
Link to this in PROD My VA Health: https://patientportal.myhealth.va.gov/pages/health_record/results/labs